creme/reports/bricks.py

```diff
@@ -34,6 +36,40 @@
 ReportGraph = reports.get_rgraph_model()
 
 
+def get_graph_colormap(graph, user):
+    if isinstance(graph.hand, RGHForeignKey):
+        fk_model = graph.hand._field.related_model
+
+        try:
+            color_field = fk_model._meta.get_field('color')
+        except Exception:
+            color_field = None
+
+        if isinstance(color_field, ColorField):
+            choices = graph.hand._abscissa_enumerator.choices(user=user)
+            fk_objects = {
+                e.pk: e for e in fk_model.objects.filter(
+                    pk__in=[c['value'] for c in choices]
+                )
+            }
+
+            return {
+                c['label']: f"#{fk_objects[c['value']].color}"
+                for c in choices
+            }
+
+    return {}
```

Comment:
Add a method to `ReportGraphHand` which returns `{}`, & an specific implementation in `RGHForeignKey`.